app/templates/components/public/speaker-item.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="speaker-item">
+
+  <img alt="{{speaker.name}}" class="ui image" src="{{if speaker.image speaker.image '/images/placeholders/avatar.png'}}">
+  <br>
+  <div class="speaker-name">{{ speaker.name }}</div>
```

Comment:
No space around the variable names. `{{speaker.name}}`. Will add all this information in the linter and a style doc soon.